sul-dlss/lyberservices-scripts
# remove assembly utils and other cleanup

* remove (now deprecated) assembly-utils gem dependency
* port methods from the utils gem that are still needed to a simple utils class within this project
* replace other functionality that existed in assembly-utils gem with inline code when it made more sense to do that (only used in one spot in the code)
* bump to latest dor-services gem (~> 7)
* update registration method to use latest dor-services-client
* update internal documentation as needed
* remove the "completion_report" script (should not be needed anymore)
* remove the "cleanup" script (destructive and should not be needed)
* remove support for the "old druid tree" directory format, as it is no longer needed
* remove support for the "cleanup" option, which is destructive
* remove support for the "reaccession" option, which can now be done better with the pre-assembly UI 
* remove associated scaffolding/cleanup methods related to removed options and scripts
* remove ruby-profiling (not working and not needed)
* remove deleted options referenced in the test setup files
* whitespace cleanup

NOTE: I'd expect some other random bits of fallout from this cleanup when it is actually exercised by Andrew, so maybe we'd want to consider deploying this branch and letting him use it
